src/main/scala/org/scalameter/Bench.scala

```diff
@@ -117,7 +120,7 @@ object Bench {
   /** A base for benchmarks generating a more detailed (regression) report,
    *  potentially online.
    */
-  abstract class HTMLReport extends Persisted[Double] {
+  abstract class HTMLReport(ctxOpt: Option[Context] = None) extends Persisted[Double] {
```

Comment:
Seems fishy from the binary compatibility standpoint. I think it would be better to introduce a secondary ctor, which passes `None`.

Btw, couldn't overriding `defaultConfig` in the concrete test achieve the same effect?
You would have to add a new `ctx: Context` argument to `Reporter`, and make sure that `defaultConfig` is passed to the `report` call in `performance-test.scala`.

Reply:
@axel22 Not sure I quite followed the suggestions regarding `Reporter` (my example test seems to work without doing this), but I was able to do the override of `defaultConfig` as suggested (which is certainly much simpler than what I had!).